services/web/server/src/simcore_service_webserver/projects/models.py

```diff
@@ -22,6 +29,43 @@ def to_project_type_db(cls, api_type: "ProjectTypeAPI") -> ProjectType | None:
         }[api_type]
 
 
+class ProjectDB(BaseModel):
+    id: str
```

Comment:
why is this not an integer? that is what I see in the database BigInt right?

Reply:
typo, thanks